test/bike_brigade_web/live/rider_live_test.exs

```diff
@@ -6,11 +6,23 @@ defmodule BikeBrigadeWeb.RiderLiveTest do
   describe "Index" do
     setup [:create_rider, :login]
 
-    test "lists all riders", %{conn: conn, rider: rider} do
-      {:ok, _index_live, html} = live(conn, ~p"/riders")
+    test "lists all riders, with working next/previous pagination", %{conn: conn} do
+      # REVIEW: is there a way I can just call :create_rider here? Do I have to import it from conn_case?
```

Comment:
It's imported already as part of `use BikeBrigadeWeb.ConnCase` 

It just expects a context map as an argument (see https://hexdocs.pm/ex_unit/1.12/ExUnit.Callbacks.html).

Note that ` use BikeBrigadeWeb.ConnCase` also imports the fixture, so you could just do `for _ <- 0..25, do: fixture(:rider)`